rpc/v2_endpoints.go

```diff
@@ -375,6 +375,12 @@ func (s *Server) GetIdentityTransfersInTickRangeV2(ctx context.Context, req *pro
 
 	}
 
+	if req.Count > 0 && len(totalTransactions) > int(req.Count) {
```

Comment:
I would move this logic a couple of lines above where `totalTransactions` is constructed